src/toil/statsAndLogging.py

```diff
@@ -231,17 +232,45 @@ def shutdown(self) -> None:
         # in addition to cleaning on exceptions, onError should clean if there are any failed jobs
 
 
-def set_log_level(level: str, set_logger: Optional[logging.Logger] = None) -> None:
+def set_log_level(level: str, set_logger: Optional[logging.Logger] = None, colored_logs: bool = False) -> None:
     """Sets the root logger level to a given string level (like "INFO")."""
     level = "CRITICAL" if level.upper() == "OFF" else level.upper()
     set_logger = set_logger if set_logger else root_logger
     set_logger.setLevel(level)
+    if colored_logs:
+        install_log_color(set_logger)
 
     # Suppress any random loggers introduced by libraries we use.
     # Especially boto/boto3.  They print too much.  -__-
     suppress_exotic_logging(__name__)
 
 
+def install_log_color(set_logger: Optional[logging.Logger] = None) -> None:
+    """Make logs colored."""
+    # Most of this code is taken from miniwdl
+    # delayed import
+    import coloredlogs  # type: ignore[import-untyped]
+
+    level_styles = dict(coloredlogs.DEFAULT_LEVEL_STYLES)
+    level_styles["debug"]["color"] = 242
+    level_styles["notice"] = {"color": "green", "bold": True}
+    level_styles["error"]["bold"] = True
+    level_styles["warning"]["bold"] = True
+    level_styles["info"] = {}
+    field_styles = dict(coloredlogs.DEFAULT_FIELD_STYLES)
+    field_styles["asctime"] = {"color": "blue"}
+    field_styles["name"] = {"color": "magenta"}
+    fmt = "%(asctime)s.%(msecs)03d %(name)s %(message)s"  # colors obviate levelname
```

Comment:
I don't think we should rely on the colors to convey level information:

1. Not everyone can actually see the difference between any given pair of colors.
2. You won't always necessarily be able to see the colors that were originally used. Maybe someone has copy-pasted the logs, or maybe you are looking at the logs on CI in the raw log view.
3. You might want to search the logs for error or warning messages, but you can't really search by color.